File: src/TouchOnline.CqrsClient/Keyboards/KeyboardForUpdate.cs
using System;
using TouchOnline.CqrsClient.Contracts;

namespace TouchOnline.CqrsClient.Keyboards
{
    public class KeyboardForUpdate : ICommand
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string LanguageCode { get; set; }
        public bool Status { get; set; }
    }
}

File: src/TouchOnline.Api/ClientApp/src/app/pages/management/keyboard-management/shared/keyboard-for-update.ts
export class KeyboardForUpdate {
    constructor(
    id?: string,
    code?: string,
    name?: string,
    languageCode?: string,
    status?: string,
    keycodesBeginners?: string) {}

    static fromJson(jsonData: any): KeyboardForUpdate {
        return Object.assign(new KeyboardForUpdate(), jsonData);
    }
}